app/parse.py

```diff
@@ -15,8 +33,110 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def init_driver(headless: bool = False) -> webdriver.Chrome:
+    options = Options()
+    options.headless = headless
+    return webdriver.Chrome(options=options)
```

Comment:
Consider adding exception handling in the `init_driver` function to manage potential errors during the WebDriver initialization, such as issues with the ChromeDriver path or browser compatibility.